Configuration/TypoScript/ContentElement/RenderChildren.typoscript

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
if you are adding file like this, I believe this should not be placed in ContentElement but in Helpers, and you should migrate at least one element to use this solution, so tt_content.shortcut for example

Reply:
Done